src/test/scala/edu/berkeley/ce/rockslicing/InputProcessingSpec.scala

```diff
@@ -5,169 +5,267 @@ import org.scalatest.FunSuite
 import scala.io.Source
 
 class InputProcessingSpec extends FunSuite {
+
+  def compareJointSets(jointSet1: JointSet, jointSet2: JointSet): Boolean = {
```

Comment:
I was wondering why `JointSet`s weren't working when I used `==` since they're supposed to automatically generate their own `Equals` method. Makes sense now based on your comments!